frontend/build/webpack.dev.conf.js

```diff
@@ -16,7 +16,7 @@ module.exports = merge(baseWebpackConfig, {
     rules: utils.styleLoaders({ sourceMap: config.dev.cssSourceMap })
   },
   // cheap-module-eval-source-map is faster for development
```

Comment:
What's the difference between `eval-source-map` and `cheap-module-eval-source-map`, and why is the comment still kicking around?

Reply:
eval-source-map is better for development.  it allows the javascript browser console to include the original code as-written with correct file names and line numbers for errors/debugging.  setting it to the previous value caused errors to reference a massive glomp of transpiled ES5 JavaScript with completely incorrect line number and no awareness of which file the original code would be in.  our computers are fast enough to do this efficiently so there is no downside.  its not appropriate for deployment, but its only set that way in the dev config.

Reply:
Awesome. can you update the comment indicating that we're switching to base source-map to from cheap-module?